src/main/java/org/gradle/profiler/ScenarioLoader.java

```diff
@@ -91,6 +94,8 @@ class ScenarioLoader {
     private static final String ANDROID_STUDIO_JVM_ARGS = "studio-jvm-args";
     private static final String ANDROID_STUDIO_IDEA_PROPERTIES = "idea-properties";
     private static final String JVM_ARGS = "jvm-args";
+    private static final String DELETE_FILE = "delete-file";
```

Comment:
❓ Do we plan to add a documentation in README.md in a separate PR? If not it would be good we add it now

Reply:
Good point.